app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -118,12 +118,14 @@ def form_params_create
   end
 
   def download_data
-    assert_privileges('cloud_object_store_container_view')
+    # TODO: rename to match others: cloud_object_store_container_view, write migration to update existing
+    assert_privileges('cloudobject_store_container_view')
     super
   end
 
   def download_summary_pdf
-    assert_privileges('cloud_object_store_container_view')
+    # TODO: rename to match others: cloud_object_store_container_view, write migration to update existing
+    assert_privileges('cloudobject_store_container_view')
```

Comment:
I don't know where in the UI to test for this.  This one is really bad.  It's a typo and should be fixed in the product features yml but that would require a migration before we can do that.  Either way, leaving it inconsistent with all of the other `cloud_object_store...` features makes it very easy to have this mistake again.

Reply:
@agrare Do you know?

Reply:
Actually @kavyanekkalapu found it...so she'll be able to verify this beyond the routes_spec.rb

<img width="1513" alt="Screen Shot 2021-05-17 at 4 11 22 PM" src="https://user-images.githubusercontent.com/19339/118665425-4d18b200-b7c0-11eb-8f3b-614504b7a050.png">

Reply:
Yes, download on this is broken for non admin users in master, i am testing this branch now.